### PR TITLE
Wallet: account screen jump to

### LINKED ISSUE #19731: Wallet account screen jump to button
- [ ] Jump to button is missing from the wallet account screen
- [ ] Jump to button colliding with the assets list items at the bottom

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/19731

### Summary
This PR adds jump to button to the wallet account screen

### Demo
https://github.com/status-im/status-mobile/assets/29354102/127c9161-e795-4b37-b9e7-6a5c796b34d8

### CODE REVIEW

**Comment by OmarBasem** [comment]
Hey @Parveshdhull, I think you are well experienced in the jump to component. 

Do you know if the above animation is correct (jumping from a `wallet-account` screen to `jump-to` page) ?

**Comment by Parveshdhull** [comment]
> Do you know if the above animation is correct (jumping from a wallet-account screen to jump-to page) ?

Hi @OmarBasem, thank you for the PR.
I think for the initial implementation it looks ok. But eventually, we probably want to have a switcher card for the wallet account and an animation of the collapsing screen on that card. (Better to confirm with the design team)

**Review by J-Son89 — DISMISSED** [review]

**Comment by OmarBasem** [comment]
Hey @Parveshdhull, could you please review this PR. 

In addition to adding the jump-to button, this PR also fixes the jump-to button colliding with the assets list items

https://github.com/status-im/status-mobile/assets/29354102/2b5e0acb-66eb-42ff-a5af-ed213661465a

File: src/status_im/contexts/wallet/account/tabs/assets/style.cljs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+(ns status-im.contexts.wallet.account.tabs.assets.style
+  (:require [status-im.contexts.shell.jump-to.constants :as constants]))
+
+(def list-container-style
+  {:padding-horizontal 8
+   :padding-bottom     constants/floating-shell-button-height})
```
Comment (Parveshdhull) [inline src/status_im/contexts/wallet/account/tabs/assets/style.cljs:6]: 🙏

**Review by Parveshdhull — APPROVED** [review]
great work @OmarBasem

**Pull request merged**